roles/imageset_mirroring/tasks/mirror-latest.yml

```
@@ -0,0 +1,59 @@
+---
+- name: Mirror latest operators
+  block:
+
+    - name: Include requirements
+      ansible.builtin.include_tasks: requirements.yml
+
+    - name: Get operator default channel and CSV
+      ansible.builtin.shell:
+        cmd: |
+          set -o pipefail
+          oc-mirror list operators \
```

This needs path to the downloaded bin:
```suggestion
          {{ im_tmp_dir }}/oc-mirror list operators \
```

OR if you accept my suggestion to:
```suggestion
          {{ _im_tmp_dir.path }}/oc-mirror list operators \
```